Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features' }.freeze
+DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features-fix' }.freeze
```

Comment:
shouldn't we merge your fix into "v0.18-dev-gencat-features"? instead of creating a new branch?

Reply:
@isaacmg410 what do you think?

Reply:
This is not a new branch. Is a new tag. As we discussed yesterday we will work with the branch system.
@aitorlb can you change the tag to the branch: `newsletter_fix_and_admin_department_refactor` please?